image/util.go

```diff
@@ -42,6 +42,16 @@ func GetImageLayerDirs(image *v1.Image) (res []string, err error) {
 	return
 }
 
+//get docker image layer hash path
+func GetBaseDockerImageLayerDir(image *v1.Image) (res []string) {
+	for _, layer := range image.Spec.Layers {
+		if layer.ID != "" && layer.Type == common.BaseImageLayerType {
+			res = append(res, filepath.Join(common.DefaultLayerDir, layer.ID.Hex()))
```

Comment:
I recommend use LayerDataDir() function under Backend to get the path, do not only const directly.